.github/workflows/blossom-ci.yml

```diff
@@ -29,13 +29,15 @@ jobs:
       args: ${{ env.args }}
 
     # This job only runs for pull request comments
-    if: contains('\
-      Nic-Ma,\
-      wyli,\
-      pxLi,\
-      YanxuanLiu,\
-      KumoLiu,\
-      ', format('{0},', github.actor)) && github.event.comment.body == '/build'
+    if: |
+      github.event.comment.body == '/build' &&
+      (
+        github.actor == 'Nic-Ma' ||
+        github.actor == 'wyli' ||
+        github.actor == 'pxLi' ||
```

Comment:
Hi @YanxuanLiu , do we still need to keep `pxLi` here for some special account?

Reply:
yes, Peixin has been back to DevOps team~